canvas-controller.js

```diff
@@ -423,6 +424,9 @@
           }
 
           if (counter >= 1 && startwaiter){
+            if starting:
+                  start = Date.now();
```

Comment:
Pretty sure this is js not python

Reply:
yeah @Skparab1 fixed it

Reply:
ye the only problem was the colon, you have to use curly braces lol